engine/src/parameters.rs

```diff
@@ -248,7 +248,7 @@ impl TryFrom<JsonValue> for NEP141FtOnTransferArgs {
         Ok(Self {
             sender_id: AccountId::try_from(value.string("sender_id")?)
                 .map_err(|_| JsonError::InvalidString)?,
-            amount: value.u128("amount")?,
+            amount: Balance::new(value.u128("amount")?),
```

Comment:
This one is right because we could be receiving any kind of NEP-141 token. You probably knew that, but I just wanted to make at least one positive comment.